### DISCUSSION TITLE
Manual

### DISCUSSION THREAD

**Discussion by AbacusQ** [deepfence/ThreatMapper#282] [General]
Is there any possibility to manually add findings? Or is this solution 100% automatically?

**Comment by shyam-dev** [comment]
What sort of findings would you like to add to the system ?

**Reply by AbacusQ** [reply]
I would like to use that system as part of a platform for penetration testing (external infrastructures, websites, API). And for now, I'm wondering if it is possible to add findings manually. To scan cloud-based services is not in my interest at the moment.

**Comment by g4dZ0r** [comment]
Is there any way to change console port?

**Reply by ramanan-ravi** [reply]
Yes, you can change [here](https://github.com/deepfence/ThreatMapper/blob/master/deployment-scripts/docker-compose.yml#L224) and [here](https://github.com/deepfence/ThreatMapper/blob/master/deployment-scripts/start-agent.sh#L13)

**Comment by g4dZ0r** [comment]
Thank you for response,

When I replace left part of “:” sign in docker-compose.yml to desired port (8123) it does not work at all. I would really appreciate if you could send me example.

Thank you in advance,
respectfully,
Rijad

**Reply by ramanan-ravi** [reply]
```
# Management console
env MGMT_CONSOLE_PORT=8123 docker-compose -f docker-compose.yml up -d

# Agent sensors
docker run -dit --cpus=".2" --name=deepfence-agent --restart on-failure --pid=host --net=host \
  --privileged=true -v /sys/kernel/debug:/sys/kernel/debug:rw -v /var/log/fenced \
  -v /var/run/docker.sock:/var/run/docker.sock -v /:/fenced/mnt/host/:ro \
  -e USER_DEFINED_TAGS="" -e MGMT_CONSOLE_URL="---CONSOLE-IP---" -e MGMT_CONSOLE_PORT="8123" \
  -e DEEPFENCE_KEY="---DEEPFENCE-API-KEY---" \
  deepfenceio/deepfence_agent_ce:1.3.1
```

**Comment by g4dZ0r** [comment]
Hello,

When I repeat same process I get Bad Gateway/dial tcp 127.0.0.1:8080: connect: connection refused error.

Thank you in advance,
Rijad
